README.md

```diff
@@ -0,0 +1,19 @@
+# Mailroom app
+
+- This app allows someone to add donors to a database and generate a thank you letter.  It also allows the user to view the list of donors in a table format, sorted by total amount donated.  The data is persisted between application runs.
+
+# Instructions
+Clone this repo, start create new virtual env, install required plugins using:
+```
+pip install -r requirements.pip
```

Comment:
If you've done the full `console_script` entry point correctly, there's a better set of instructions for installation:

```
pip install .[test]
```

And usage:

```
bin/mailroom
```

If this doesn't work, then your setup.py file is not correctly configured.